templates/include/glean/glean.1.schema.json

```diff
@@ -381,6 +381,30 @@
           }
         }
       ]
+    },
+    "experiments": {
+      "propertyNames": {
+        "$ref": "#/definitions/short_id"
+      },
+      "additionalProperties": {
+        "type": "array",
+        "minItems": 1,
```

Comment:
Now this is interesting, can you elaborate on what the extra field is for? This does seem like a non-intuitive structure for this layout. It might be better to support it as e.g.:
```
[
  {
    "experiment_id": " experiment2",
    "experiment_branch": "branch_b",
    "extras: {
       "extra_key": "extra_val"
    }
]
```

But it really depends on what the `extra` part is for.

Reply:
I'm not at all opposed to making each of these entries an object.

"extras" is the same as "options" on desktop (renamed due to confusion).  It's basically a place for experiment authors to put extra metadata about the experiment settings, I suppose if a branch name alone isn't enough.

Reply:
I'm not familiar with "options" on desktop, can you point me to that? All I see are `experiment_id` and `experiment_branch` [0].

[0] http://gecko-docs.mozilla.org.s3.amazonaws.com/toolkit/components/telemetry/telemetry/data/environment.html

Reply:
Ah, I think this might be based on my misunderstanding.  I was looking at this:

https://searchfox.org/mozilla-central/rev/efc0d9172cb6a5849c6c4fc0f19d7fd5a2da9643/toolkit/components/telemetry/app/TelemetryEnvironment.jsm#85-115

But if you drill down into the implementation, it looks like all it stores to send to the backend is `options.type`, if it exists.

https://searchfox.org/mozilla-central/source/toolkit/components/telemetry/app/TelemetryEnvironment.jsm#1018

I'm leaning to just taking that all out, but I'll bring it to a discussion first.